ddtrace/tracer.py

```diff
@@ -136,6 +138,7 @@ def _default_span_processors_factory(
     single_span_sampling_rules,  # type: List[SpanSamplingRule]
     agent_url,  # type: str
     profiling_span_processor,  # type: EndpointCallCounterProcessor
+    peer_service_enabled,  # type: bool
```

Comment:
I see a few patters for including post-processors here, and I wasn't sure if any where preferred, so I just picked one.  Let me know if I should use a different method!

Reply:
A SpanProcessor feels a bit odd here since these tags will be an important part of the span data model but I don't have a better suggestion. Maybe we can move this logic to [tracer.trace(...)](https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/tracer.py#L813) but I am not a big fan of adding more logic to that complex method. 

Just rambling I guess. A SpanProcessor works for me as long as there is not a significant performance impact.

Reply:
The other mechanism I came up with for this is updating every contrib to add the tags, which is implementation and testing heavy